backend/experiment/rules/gold_msi.py

```diff
@@ -40,3 +40,8 @@ def next_round(self, session, request_session=None):
             return questions
         else:
             return final_action_with_optional_button(session, '', request_session)
+
+    def feedback_info(self):
+        info = super().feedback_info()
+        info['show_float_button'] = True
+        return info
```

Comment:
Hmm, we have existing experiment series that have the `gold_msi` rules included. It would be weird to include the floating button there but not in the other experiments. So perhaps remove this from this specific experiment.